projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -688,6 +688,7 @@ public static function enqueue_block_editor_assets() {
 			$is_current_user_connected = ( new Connection_Manager( 'jetpack' ) )->is_user_connected();
 		}
 
+		wp_enqueue_style( 'recoleta-font', '//s1.wp.com/i/fonts/recoleta/css/400.min.css', array(), Constants::get_constant( 'JETPACK__VERSION' ) );
```

Comment:
Recoleta doesn't seem to be used in Jetpack. I'd argue it's overkill to load a new font for a heading.

Reply:
I also though about this as well but Recoleta is used on .com as the very common element font when we display banners about the Jetpack app. I will update the code so that we don't load it if we don't need it for now. 
and revisit this decision later.